docs/architecture/adr-013-non-interactive-default-rules-for-zero-padding.md

```diff
@@ -2,57 +2,69 @@
+Implemented in <https://github.com/celestiaorg/celestia-app/pull/1604>. `SubtreeRootThreshold` decreased from 128 to 64 in <https://github.com/celestiaorg/celestia-app/pull/1766>.
 
 ## Changelog
 
 - 2023/03/01: Initial draft
 - 2023/05/30: Update status
+- 2023/06/17: Add a section for "Picking a `SubtreeRootThreshold`". Rephrased some sections.
 
 ## Context
 
-When laying out blobs in the square we create padding to follow the non-interactive default rules. With ADR 009 we decreased padding by aligning the blob in the square to the index of the `MinSquareSize` of the given blob. This is a good improvement but we can do better.
+When laying out blobs in the square, the block proposer adds namespace padding shares between blobs to conform to the non-interactive default rules. [ADR-009](./adr-009-non-interactive-default-rules-for-reduced-padding.md) reduced padding by placing blobs at an index that is a multiple of the `MinSquareSize(blob)`[^1]. This is a good improvement but we can reduce padding further.
 
-Looking at different ranges of blob sizes we can see that the ratio of blob size to padding is not constant. Insight:  **The ratio of blob size to padding is smaller for smaller blobs and larger for larger blobs.** The bigger the ratio the better.
+If we analyze the worst case padding for different blob sizes, we can see that the ratio of blob size to padding is not constant. Insight: **the ratio of blob size to padding is small for small blobs and large for large blobs.**
 
 ![Worst Case Padding In Blob Size Range](./assets/adr013/worst-case-padding-in-blob-size-range.png)
 
-This means small blobs generate more possible padding in comparison to the data they provide. This is not ideal as we want to have as little padding as possible. As padding is not being paid for there is no incentive to use larger blobs.
+This means small blobs are inefficient because they generate more potential padding for the data they provide. This is not ideal as we want to minimize padding. Since users do not pay for namespace padding shares, they may not be sufficiently incentivized to submit large blobs.
 
-In the naive approach if you align the blob at an index of one you get zero padding as each blob can follow the next. This would make the hash of each share a subtree root. In a square with N shares, you would always get N subtree roots. But having a blob inclusion proof of size N for large blobs is too much and unfeasible.
+In the naive approach if the block proposer aligned blobs one after another then there would be zero padding between blobs. Although this approach minimizes padding, it comes at the cost of large blob inclusion proofs because the hash of each share would be a subtree root of the proof. In a square with N shares, there would be N subtree roots. As a result, light nodes would need to download N subtree roots to verify blob inclusion proofs for this square which may be too large for resource constrained devices.
```

Comment:
In the last sentence, it is necessary to say that the blob inclusion proof for **a blob of size N** would include N subtree roots. Light Nodes don't have to download N subtree roots if they only need the inclusion of a small blob.

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1959/commits/4a4d33628a8d8c6f1f7664f34d18d7c4e9d77830